InventoryIndexer/Model/ReservationDummy.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryIndexer\Model;
+
+/**
+ * ----
+ */
+class ReservationDummy
+{
+    /**
+     * @param string[] $array
+     * @return void
+     */
+    public function execute($array)
+    {
+        return true;
```

Comment:
you may accept payload here and call `IsProductSalable` service.
That's what we need to have in indexer anyways